core/src/main/java/org/apache/carbondata/core/statusmanager/SegmentUpdateStatusManager.java

```diff
@@ -104,9 +102,29 @@ public SegmentUpdateStatusManager(CarbonTable table) {
     } else {
       updateDetails = new SegmentUpdateDetails[0];
     }
+    updateUpdateDetails(updateVersion);
     populateMap();
   }
 
+  private void updateUpdateDetails(String updateVersion) {
+    if (updateVersion != null) {
+      List<SegmentUpdateDetails> newupdateDetails = new ArrayList<>();
```

Comment:
please give comment to describe the functionality of this func

Reply:
ok